### PR TITLE
cli: minor docs changes for 8.0.0 release

### PR DESCRIPTION
**Pull request by oliver-sanders**
Had a quick review of the CLI docs, found a couple of small things.

Reformatted a couple of paragraphs which were unnecessarily "narrow" due to being copied from the schema docs?

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Applied any dependency changes to both `setup.cfg` and `conda-environment.yml`.
- [x] Does not need tests (why?).
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.

### CODE REVIEW

**Review by wxtim — APPROVED** [review]

File: cylc/flow/scripts/check_versions.py [hunk 1]
```diff
@@ -20,17 +20,17 @@
...
 
-It is recommended that cylc versions be installed in parallel and access
-configured via the cylc version wrapper as described in the cylc INSTALL
-file and User Guide. This must be done on workflow and task hosts. Users then
-get the latest installed version by default, or (like tasks) a particular
-version if $CYLC_VERSION is defined.
+Note:
+  Cylc supports multiple parallel installations at different versions via the
+  Cylc wrapper script. For more information see the installation section of the
+  documemtation.
```
Comment (MetRonnie) [inline cylc/flow/scripts/check_versions.py:30]:
```suggestion
  documentation.
```

File: cylc/flow/scripts/check_versions.py [hunk 2]
```diff
@@ -20,17 +20,17 @@
 
 Check that Cylc versions match on different platforms.
 
-Check the version of cylc invoked on each of WORKFLOW's task host platforms
-when CYLC_VERSION is set to *the version running this command line tool*.
+Check the version of Cylc invoked on each of workflow's task host platforms
+when CYLC_VERSION is set to <bold>the version running this command line
+tool</bold>.
```
Comment (MetRonnie) [inline cylc/flow/scripts/check_versions.py:25]: I am a bit unclear on what it means by "check the version... when CYLC_VERSION is set to..."

File: cylc/flow/scripts/validate.py [hunk 3]
```diff
@@ -20,9 +20,9 @@
 
 Validate a workflow configuration.
 
-If the workflow definition uses include-files reported line numbers
-will correspond to the inlined version seen by the parser; use
-'cylc view -i,--inline WORKFLOW' for comparison.
+If the workflow definition uses include-files reported line numbers will
```
Comment (MetRonnie) [inline cylc/flow/scripts/validate.py:23]:
```suggestion
If the workflow definition uses include-files, reported line numbers will
```

File: cylc/flow/scripts/check_versions.py (hunk 2 above)
Comment (oliver-sanders) [reply cylc/flow/scripts/check_versions.py:25]: It is checking that the remote versions are set to the local version as defined via CYLC_VERSION

Comment (oliver-sanders) [reply cylc/flow/scripts/check_versions.py]: Reworded.

**Pull request merged**